src/components/ButtonReset/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import './button-reset.css';
+const ButtonReset = (props) => {
+    return (
+        <button className="button-reset" onClick={ props.handleClick }>reset</button>
```

Comment:
You can use destructuring. For example: instead of `(props)` write `({ handleClick })` and then `onClick={ handleClick }`.